editor.planx.uk/src/pages/FlowEditor/components/FeedbackLog/FeedbackLog.tsx

```diff
@@ -96,13 +96,9 @@ export const FeedbackLog: React.FC<FeedbackLogProps> = ({ feedback }) => {
       field: "nodeType",
       headerName: "Where",
       width: 280,
-      type: ColumnFilterType.CUSTOM,
```

Comment:
This has been reverted to a default (string) column type allowing string filtering (contains, does not contain etc). We could change this to be a `SINGLE_SELECT` again (based on just node type, not node title). Again open to ideas on what's best - this seems to offer the most flexibility just now.